lib/blkdev.c

```diff
@@ -56,25 +59,33 @@ int is_blkdev(int fd)
 
 off_t
 blkdev_find_size (int fd) {
-	uintmax_t high, low = 0;
+	intmax_t high = 1024, low = 0;
```

Comment:
What about using directly off_t (rather than intmax_t)? This code will never be able to work with anything bigger as the core of this functionality is sleek().

Reply:
That's true. I have seen the SINT_MAX macro in another pull request, so I have adjusted the commit to use it.